0.3.5 fails to build with `--no-default-features --features cgemm`

The errors look like this:

```
cargo build --no-default-features --features cgemm
   Compiling matrixmultiply v0.3.5 (/home/autarch/projects/matrixmultiply)
error[E0433]: failed to resolve: use of undeclared crate or module `std`
  --> src/cgemm_common.rs:10:5
   |
10 | use std::ptr::copy_nonoverlapping;
   |     ^^^ use of undeclared crate or module `std`

error[E0432]: unresolved import `std`
 --> src/cgemm_common.rs:9:5
  |
9 | use std::mem;
  |     ^^^ use of undeclared crate or module `std`

error[E0599]: no method named `mul_add` found for type `f32` in the current scope
   --> src/cgemm_common.rs:21:23
    |
21  |               $dst = $a.mul_add($b, $dst);
    |                         ^^^^^^^ method not found in `f32`
    |
   ::: src/cgemm_kernel.rs:199:1
    |
199 | / kernel_fallback_impl_complex! {
200 | |     // instantiate separately
201 | |     [inline target_feature(enable="avx2") target_feature(enable="fma")] [fma_yes]
202 | |     kernel_target_avx2, T, TReal, KernelAvx2::MR, KernelAvx2::NR, 4
203 | | }
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `fmuladd` which comes from the expansion of the macro `kernel_fallback_impl_complex` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0599]: no method named `mul_add` found for type `f64` in the current scope
   --> src/cgemm_common.rs:21:23
    |
21  |               $dst = $a.mul_add($b, $dst);
    |                         ^^^^^^^ method not found in `f64`
    |
   ::: src/zgemm_kernel.rs:196:1
    |
196 | / kernel_fallback_impl_complex! {
197 | |     // instantiate fma separately
198 | |     [inline target_feature(enable="fma") target_feature(enable="avx2")] [fma_yes]
199 | |     kernel_target_avx2, T, TReal, KernelAvx2::MR, KernelAvx2::NR, 4
200 | | }
    | |_- in this macro invocation
    |
    = note: this error originates in the macro `fmuladd` which comes from the expansion of the macro `kernel_fallback_impl_complex` (in Nightly builds, run with -Z macro-backtrace for more info)

Some errors have detailed explanations: E0432, E0433, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `matrixmultiply` due to 14 previous errors
```

--- a/ensure_no_std/Cargo.toml
+++ b/ensure_no_std/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 publish = false
 
 [dependencies]
-matrixmultiply = { path = "..", default-features = false }
+matrixmultiply = { path = "..", default-features = false, features = ["cgemm"] }
 
 [profile.dev]
 panic = "abort"

--- a/src/cgemm_common.rs
+++ b/src/cgemm_common.rs
@@ -6,14 +6,15 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use std::mem;
-use std::ptr::copy_nonoverlapping;
+use core::mem;
+use core::ptr::copy_nonoverlapping;
 
 use rawpointer::PointerExt;
 
 use crate::kernel::Element;
 use crate::kernel::ConstNum;
 
+#[cfg(feature = "std")]
 macro_rules! fmuladd {
     // conceptually $dst += $a * $b, optionally use fused multiply-add
     (fma_yes, $dst:expr, $a:expr, $b:expr) => {
@@ -22,6 +23,15 @@ macro_rules! fmuladd {
         }
     };
     (fma_no, $dst:expr, $a:expr, $b:expr) => {
+        {
+            $dst += $a * $b;
+        }
+    };
+}
+
+#[cfg(not(feature = "std"))]
+macro_rules! fmuladd {
+    ($any:tt, $dst:expr, $a:expr, $b:expr) => {
         {
             $dst += $a * $b;
         }

